source/api_cc/src/DeepPotJAX.cc

```
@@ -0,0 +1,581 @@
+}
+
+inline TF_DataType get_data_tensor_type(const std::vector<int32_t>& data) {
+  return TF_INT32;
+}
+
+inline TF_DataType get_data_tensor_type(const std::vector<int64_t>& data) {
+  return TF_INT64;
+}
+
+inline TFE_Op* get_func_op(TFE_Context* ctx,
+                           const std::string func_name,
```

cppcheck

[performance] Function parameter 'func_name' should be passed by const reference.

(passedByValue)